packages/frontend/src/features/sqlRunner/store/tableVisSlice.ts

```diff
@@ -41,9 +41,9 @@ export const tableVisSlice = createSlice({
         builder.addCase(setSqlRunnerResults, (state, action) => {
             if (state.config === undefined) {
                 // TODO: this should come from the transformer
-                const columns = Object.keys(action.payload.results[0]).reduce<
-                    TableChartSqlConfig['columns']
-                >(
+                const columns = Object.keys(
+                    action.payload.data.results[0],
```

Comment:
does the comma do anything different here? odd eslint+formatting 😅

Reply:
I dont think so. I think pretty print broke the line and always adds a comma in that case